packages/git/src/browser/git-preferences.ts

```diff
@@ -66,7 +66,9 @@ export function createGitPreferences(preferences: PreferenceService): GitPrefere
 export function bindGitPreferences(bind: interfaces.Bind): void {
     bind(GitPreferences).toDynamicValue(ctx => {
         const preferences = ctx.container.get<PreferenceService>(PreferenceService);
-        return createGitPreferences(preferences);
+        const prefs = createGitPreferences(preferences);
+        prefs["git.editor.dirtyDiff.linesLimit"] = '20';
+        return prefs;
```

Comment:
@sdirix are these changes related, they are currently failing the build.

Reply:
I committed this file by mistake. I'll remove it again.